OrganizedToDoListApplication.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ */
+
+package edu.brooklyncollege.cisc3130.project.organizedtodolistapplication;
+
+/* this main method class handles user input for OrganizedToDoListApplication 
+ */
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+public class OrganizedToDoListApplication {
+    public static void main(String[] args) {
+        TaskList<Task<String>> taskList = new TaskList<>();
+        Scanner in = new Scanner(System.in);
+        String keyWord;
+
+        System.out.println("You have joined a text-based ToDoList application");
+
+        while (true) {
+            //this is something how it would look like 
+            //System.out.println("\nEnter one of these key words: add, addSpecificIndex, remove, move, sort, display, exit");
+            System.out.println("\nEnter add or exit as a key word");
```

Comment:
You can keep all supported keywords in a set and use that to generate the message of what they can enter